app/controllers/patients_controller.rb

```diff
@@ -23,7 +23,7 @@ def show
 
     @jurisdiction = @patient.jurisdiction
     @laboratories = @patient.laboratories.order(:created_at)
-    @close_contacts = @patient.close_contacts.order(:created_at)
+    @close_contacts = @patient.close_contacts.where(archived: [false, nil]).order(:created_at)
```

Comment:
```suggestion
    @close_contacts = @patient.close_contacts.where.not(archived: true).order(:created_at)
```

Reply:
The sql query this generates doesn't collect close contacts where the value is nil since NULL is treated differently in SQL backend, which would be all close contacts made before the change goes into effect.